spec/requests/queues/objects_for_step_spec.rb

```diff
@@ -91,17 +91,34 @@
                           active_version: true)
       end
 
-      it 'shows the items that are waiting and have met the prereqs' do
-        get '/workflow_queue?waiting=dor%3AaccessionWF%3Areset-workspace&' \
-            'completed=dor%3AaccessionWF%3Asdr-ingest-received&' \
-            'completed=dor%3AaccessionWF%3Aprovenance-metadata&limit=100&lane-id=default'
+      context 'with repository-qualified step names' do
+        it 'shows the items that are waiting and have met the prereqs' do
+          get '/workflow_queue?waiting=dor%3AaccessionWF%3Areset-workspace&' \
+              'completed=dor%3AaccessionWF%3Asdr-ingest-received&' \
+              'completed=dor%3AaccessionWF%3Aprovenance-metadata&limit=100&lane-id=default'
+
+          expect(response.body).to be_equivalent_to <<~XML
+            <objects count="2">
+              <object id="#{prereqs_and_waiting.druid}"/>
+              <object id="#{second_prereqs_and_waiting.druid}"/>
+            </objects>
+          XML
```

Comment:
ditto on expect_xml as memoized var?  and other places below.

Reply:
For sure.